monai/data/utils.py

```diff
@@ -547,6 +548,30 @@ def set_rnd(obj, seed: int) -> int:
     return seed
 
 
+def affine_to_spacing(affine: NdarrayTensor, dims: int = 3, dtype=float, suppress_zeros: bool = True) -> NdarrayTensor:
```

Comment:
Hi @wyli ,

I think this is the function used in: https://github.com/Project-MONAI/MONAI/issues/3766, right?

Thanks.

Reply:
Yes, this commit  is to address it,  commit message links to it https://github.com/wyli/MONAI/commit/b32336e596009127c46c813149296fa835697471